src/ContentHubClient.php

```diff
@@ -328,51 +331,12 @@ public function getEntities(array $uuids)
      */
     protected function getCDFObject($data)
     {
-        $event = new GetCDFTypeEvent($data);
+        $event = ObjectFactory::getCDFTypeEvent($data);
         $this->dispatcher->dispatch(ContentHubLibraryEvents::GET_CDF_CLASS, $event);
 
         return $event->getObject();
     }
 
-  /**
-   * Updates an entity asynchronously.
-   *
-   * The entity does not need to be passed to this method, but only the resource URL.
-   *
-   * @param \Acquia\ContentHubClient\CDF\CDFObject $object
-   *   The CDFObject
-   *
-   * @return \Psr\Http\Message\ResponseInterface
-   *
-   */
-    public function putEntity(CDFObject $object)
-    {
-        $options['body'] = json_encode(['entities' => [$object->toArray()], 'resource' => ""]);
-        return $this->put("entities/{$object->getUuid()}", $options);
-    }
-
-  /**
-   * Updates many entities asynchronously.
-   *
-   * @param \Acquia\ContentHubClient\CDF\CDFObject[] $objects
-   *   The CDFObjects to update.
-   *
-   * @return \Psr\Http\Message\ResponseInterface
-   *
-   */
-    public function putEntities(CDFObject ...$objects)
```

Comment:
\Drupal\acquia_contenthub_publisher\Plugin\QueueWorker Line 171 actually does a putEntities(). I think it's postEntities we don't use.

Reply:
I added it back!